examples/golang/verify.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+export NAME=golang
+export UID
+export DELAY=5
```

Comment:
i think this is not necessary - probably we want to set:

```bash
export MANUAL=true
```

and then later, once the `.so` has been built you can do

```bash
bring_up_example
```

Reply:
Done. It's good for me.